docs/_posts/2017-06-13-react-v15.6.0.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "React v15.6.0"
+author: flarnie
+---
+
+Today we are releasing React 15.6.0. As we prepare for React 16.0, we have been fixing and cleaning up many things. This release continues to pave the way.
+
+## Improving Inputs
+
+In React 15.6.0 the 'onChange' event for inputs is a little bit more reliable and handles more edge cases, including the following:
```

Comment:
Let's use backticks, not apostrophes. Backticks appear as code blocks in the post.